### PR TITLE
Add declarations for "svg-path-bounds"

### PR DESCRIPTION
**Pull request by larsmunkholm**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

**Comment by peterblazejewicz** [comment]
> * [x]  The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
> *

are you sure about this? The package seems to be shippings it's own declaration:
https://github.com/dy/svg-path-bounds/blob/master/package.json#L6

**Comment by larsmunkholm** [comment]
@peterblazejewicz I know. I'm the one who added them. But the owner of the repo isn't publishing to npm or responding to messages.

File: types/svg-path-bounds/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+// Type definitions for svg-path-bounds 1.0
+// Project: https://www.npmjs.com/package/svg-path-bounds
+// Definitions by: larsmunkholm <https://github.com/larsmunkholm/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export = svgPathBounds;
+
+declare function svgPathBounds(path: string): [number, number, number, number];
```
Comment (peterblazejewicz) [inline types/svg-path-bounds/index.d.ts:8]:
```suggestion
declare function svgPathBounds(path: string): [left: number, top: number, right: number, bottom: number];
```
this is optional, named tuples support is now wide (TS4.0):
https://www.typescriptlang.org/play?ts=4.0.2#example/named-tuples

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@larsmunkholm thanks!

**Comment by larsmunkholm** [comment]
Ready to merge

**Pull request merged**